docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/risk_exposure/index.rst

```diff
@@ -7,3 +7,172 @@ Iron Deficiency Risk Exposure
+
+	Where, :math:`\sum_{c=1}^{n} p_c * shift_c` represents the sum of the cause-specific prevalence multiplied by the cause-specific hemoglobin shift for all causes of iron deficiency and :math:`mean_\text{Hgb}` is the observed mean population hemoglobin concentration (MEID 10487).
+
+	An estimation of the TMREL using this method can be found `here <https://github.com/ihmeuw/sim_sci_maternal_anemia/blob/master/data_validation/maternal_disorders_burden/custom_paf_calculations.ipynb>`_.
+
+	The cause-specific hemoglobin shifts represent the average difference in hemoglobin concentration among individuals afflicted with that cause and the general population (from the literature) and can be found `here <https://github.com/ihmeuw/sim_sci_maternal_anemia/blob/master/data_validation/maternal_disorders_burden/hb_shifts.csv>`_. Note that these shifts are sex-specific but do not vary by age. Additionally, these shifts apply to mean hemoglobin concentration only; it is assumed that there is no change in hemoglobin distribution standard deviation, which is a limitation in this analysis due to a significant data gap.
+
+	The following table lists the relevant IDs for the diseases that cause iron deficiency that should be used for this analysis.
+
+.. list-table:: IDs Used for the Location-Specific TMREL exposure estimation
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Disease
+    - ID Type
+    - ID
+  * - Stage 5 Chronic Kidney Disease
+    - MEID
+    - 2022
+  * - Stage 4 Chronic Kidney Disease
+    - MEID
+    - 10733
+  * - Stage 3 Chronic Kidney Disease
+    - MEID
+    - 10732
+  * - Symptomatic Uterine Fibroids
+    - MEID
+    - 3121
+  * - Menstrual Disorders
+    - MEID
+    - 18741
+  * - Crohn's Disease
+    - MEID
+    - 3104
+  * - Ulcerative Colitis
+    - MEID
+    - 3103
+  * - Maternal Hemorrhage Adjusted for Live Births
+    - MEID 
+    - 3620
+  * - Cirrhosis
+    - Cause
+    - 521
+  * - Gastritis
+    - Cause
+    - 528
+  * - Peptic Ulcer Disease
+    - Cause
+    - 527
+  * - Other endocrine, nutrition, blood, and immune disorders
+    - Cause
+    - 619
+  * - Other Infectious Diseases
+    - Cause
+    - 408
+  * - Hookworm Disease
+    - Cause
+    - 363
+  * - Other Neglected Tropical Diseases
+    - Cause
+    - 365
+  * - Schistosomiasis
+    - Cause
+    - 351
+  * - Dietary Iron Deficiency
+    - Cause
+    - 390
+  * - Vitamin A Deficiency
+    - Cause
+    - 389
+  * - End-Stage Renal Disease on Dialysis
+    - Sequela
+    - 22989, 22990, 22991, 22992, 22993, 22999, 23000, 23001, 23002, 23003, 23009, 23010, 23011, 23012, 23013, 23019, 23020, 23021, 23022, 23023
+
+.. note::
+
+	There are some diseases that cause iron deficiency that are available in the necessary format as modelable entities or sequelae rather than causes, which is why several ID types are listed in the table above.
+
+#. Estimate iron deficiency risk exposure as the observed mean population hemoglobin concentration
```

Comment:
This second step is also numbered "1." in the rendered version. Maybe because your list-table and note are not indented to the same level as your numbered list?